docs/ide4c/vscode/tasks.json

```diff
@@ -0,0 +1,41 @@
+{
+    "version": "2.0.0",
+    "tasks": [
+        {
+            "label": "Makefile", // You can choose your own label.
+            "type": "shell",
+            "command": "make",
+            // Make this the default build command.
+            "group": {
+                "kind": "build",
+                "isDefault": true
+            },
+            "presentation": {
+                "reveal": "always"
+            },
+            // You can pass all the building arguments here.
+            "args": [
+                "all",
+                "--debug", // Enable debug mode.
+                "-j8" // Use 8 cores.
+            ]
+        },
+        {
+            "label": "buildTest", // for building test cases before running it.
+            "command": "make", // Could be any other shell command
+            "args": [
+                "-j16",
+                "-C",
+                "${workspaceFolder}/test/intltest"
+            ],
+            "type": "shell"            
+        },
+        {
+            "label": "runTestCases", // For excuting the test cases.
+            "command": "${workspaceFolder}/test/intltest/intltest", // Could be any other shell command
+            "args": [
```

Comment:
Please give an example for how you would specify a single test.  For example,

```js
"args": [
    // Specify a specific test case here; for example:
    // "format/NumberTest"
]
```

Reply:
This is in the launch task.